mscore/musescore.cpp

```diff
@@ -2816,7 +2817,16 @@ bool MuseScore::hasToCheckForUpdate()
       if (ucheck)
             return ucheck->hasToCheck();
       else
-          return false;
+            return false;
+      }
+
+//---------------------------------------------------------
+//   hasToCheckForPackagesUpdate
```

Comment:
It's called Extension all over the place. Why package here ?

Reply:
Typo :)